packages/web/src/components/pages/Package.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Error } from 'components/layouts';
+import PackagePage from '../layouts/PackagePage/PackagePage';
+import { useParams } from 'react-router-dom';
+import { usePackageInfo } from '../../store/hooks/usePackageInfo';
+import { Helmet } from 'react-helmet';
+
+export function PackageView() {
+  const { '*': packageName } = useParams();
+  if (!packageName) {
+    return <Error host='' />; // TODO
+  }
+
+  const { packageInfo, isLoading, error } = usePackageInfo(packageName) ?? {};
+
+  if (packageName && !isLoading && (error || !packageInfo)) {
+    // TODO: 404 error page
+    return <Error host={packageName} message='Package is not found in the database' />;
+  }
+
+  const title = `Real world usage & stats of ${packageName} - GradeJS`;
+  const description =
+    `GradeJS has discovered ${packageInfo?.usage.length} websites using ${packageName}` +
+    (packageInfo?.usage.length
```

Comment:
So currently `usage` is capped to 16. We probably need a separate `count` query.

Reply:
Also, please create an issue -- we need some kind of `og:image` here.

Reply:
I guess we can use the default one for now